packages/jest-leak-detector/src/__tests__/index.test.ts

```diff
@@ -84,21 +95,30 @@ it('correctly checks more complex leaks', () => {
   const detector1 = new LeakDetector(ref1);
   const detector2 = new LeakDetector(ref2);
 
+  let isLeaking1: boolean;
```

Comment:
Is there a reason for extracting these to variables? It seems to add a lot of extra noise

Reply:
yep, if you use code below:
```js
expect(await detector1.isLeaking()).toBe(true); 
```
param received in expect not is result, it's will fail.